COMMUNITY-TEMPLATES.md

```diff
@@ -0,0 +1,59 @@
+# Using Graphback With Your Own Templates
+
+In case you want to use some other templates than the ones offered, Graphback also supports adding your templates. The process is fairly simple and is as follows:
+
+## Step 1: Creating an object for your template
+
+To add your template, you first need to have it in the form of an object which you'll later add to an array. This is best explained by an example:
+
+Let's say you have a template which you want to name `a-sample-template` and the files for it are located in the repository `https://github.com/sample-user/sample-project`. Then the object for your template would look like this:
+
+```
```

Comment:
This should be ts file

Reply:
Do you mean the url should point to a particular ts file on the person's repository?